Source/FFMPEGMedia/Private/Player/FFMPEGMediaTracks.cpp

```diff
@@ -2546,8 +2546,8 @@ int FFFMPEGMediaTracks::AudioRenderThread() {
             RenderAudio();
             int64_t endTime =  av_gettime_relative();
             int64_t dif = endTime - startTime;
-            if ( dif < 33333) {
-                av_usleep(33333 - dif);
+            if ( dif < 32333) {

```

Comment:
I'm not quite sure about the FPS, we've tested a lot of different videos and all had the same behaviour which was fixed after lowering that value (we also tested RTMP streams). 

I guess this value should indeed be calculated by the video FPS since it also drives the playback speed when the Time control is set to audio.